source/03-components/tabs/tabs.scss

```diff
@@ -11,8 +11,9 @@
   overflow-x: auto;
   padding-inline: gesso-site-margins(mobile);
   position: sticky;
-  top: var(--gesso-header-current-height, 0);
+  top: calc(var(--gesso-header-current-height, 0) + var(--ginToolbarHeight, 0));
```

Comment:
@jackmakesthings One counter-intuitive thing here: when using `calc()`, we need to change the fallback to `0px` as calc won't work with the unitless zero we use elsewhere. (https://www.oddbird.net/2022/08/04/zero-units/ goes into it in more detail; it has to do with what's considered a number vs a length)